textproc/pandoc/Portfile

```diff
@@ -1,90 +1,50 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           haskell_stack 1.0
 
-categories          textproc haskell
+github.setup        jgm pandoc 2.11.4
+revision            0
+categories-append   haskell
```

Comment:
please revert to the original as `port lint` points out

Reply:
Thanks done.